run-local.sh

```diff
@@ -27,12 +27,21 @@ export DB_PASS=create-and-vary-a-licence
 
 # Provide URLs to other local container-based dependent services
 # Match with ports defined in docker-compose.yml
-export HMPPS_AUTH_URL=http://localhost:9090/auth
+export HMPPS_AUTH_URL=https://sign-in-dev.hmpps.service.justice.gov.uk/auth
 
 # Make the connection without specifying the sslmode=verify-full requirement
 export SPRING_DATASOURCE_URL='jdbc:postgresql://${DB_SERVER}/${DB_NAME}'
 
+# Bring up the licences-db container needed for the application
+docker-compose down --remove-orphans && docker-compose pull
+docker-compose -f docker-compose.yml up -d
```

Comment:
should we use `docker compose` now? 
don't need to specify file as that is the default

Reply:
Doh - it's not as though I just posted about this very thing a couple of hours earlier!

Reply:
meh... do